operations/network/network_management_install_guide/manual_switch_config.md

```diff
@@ -0,0 +1,49 @@
+# Manual switch configuration
+
+Some switch configuration is not generated by CANU and needs to be manually applied.
+
+#### SNMP configuration
+SNMP is currently only used on sw-leaf-bmc switches, these credentials can be retrieved from vault.  More info on SNMP creds can be found on the [Change SNMP Credentials on Leaf Switches](../../operations/security_and_authentication/Change_SMNP_Credentials_on_Leaf_Switches.md) page.
```

Comment:
Path is broken. Should be 
../../../security_and_authentication/Change_SMNP_Credentials_on_Leaf_Switches.md